Explorer/Assets/DCL/Profiles/Repository/RealmProfileRepository.cs

```diff
@@ -60,6 +62,11 @@ public async UniTask SetAsync(Profile profile, CancellationToken ct)
                 await ipfs.PublishAsync(entity, ct, files);
                 profileCache.Set(profile.UserId, profile);
             }
+            catch (Exception e)
+            {
+                Debug.LogError("Could not save the profile");
```

Comment:
Please use `ReportHub`.

Reply:
Dont worry, will revert this change, its a wip